test/src/index/transport/testBrowserstackTransport.js

```diff
@@ -65,6 +65,32 @@ describe('BrowserstackTransport', function () {
     delete process.env['KEY'];
   });
 
+  it('test create Transport for Browserstack - empty buildName', function() {
+    process.env['USER'] = 'test-access-user';
+    process.env['KEY'] = 'test-access-key';
+
+    const client = NightwatchClient.client({
+      webdriver: {
+        host: 'hub-cloud.browserstack.com',
+        port: 443
+      },
+      desiredCapabilities: {
+        'browserstack.user': '${USER}',
```

Comment:
you can just set the value here directly.

Reply:
Here, I wanted to see if we were setting buildName or not when buildName was missing from the settings.